### DISCUSSION TITLE
Multiple Harvesters spams WALLET unhealthy errors & Wallet/Farmer only node spams HARVESTER errors

### DISCUSSION THREAD

**Discussion by frntman** [martomi/chiadog#389] [General]
I recently upgraded my chiadog from a very early version to current and now it's unusable because I can't figure out how to configure the notifications? I run wallet/farmer on one node and multiple harvesters that have individual chiadog instances (just learned that the harvesters can be monitored from a single remote location) however I'm I missing something in the config or is there a way to revert to earlier version? 

thanks :-)

**Comment by bradyjarvis** [comment]
You probably just need to edit your current config to look like the new one.  Take a look at the config-example.yaml.  If I recall, I backed up my old config file by renaming it to config.old or something, and then did `cp config-example.yaml config.yaml`.  Then I edited the new config.yaml to have the values from my specific setup.

**Reply by frntman** [reply]
Thank You !!! I did look at the config example and totally missed the new stuff that had been added and most importantly, the monitored services section.
